site/content/en/docs/tasks/run_python_jobs.md

```diff
@@ -0,0 +1,241 @@
+---
+title: "Run Jobs Using Python"
+date: 2023-07-05
+weight: 7
+description: >
+  Run Kueue jobs programmatically with Python
+---
+
+This guide is for [batch users](/docs/tasks#batch-user) that have a basic understanding of interacting with Kueue from Python. For more information, see [Kueue's overview](/docs/overview).
```

Comment:
```suggestion
This guide is for [batch users](/docs/tasks#batch-user) that have a basic understanding of interacting with Kubernetes from Python. For more information, see [Kueue's overview](/docs/overview).
```
?

Interacting with Kueue from Python is rather what this guide is about and shouldn't be a pre-requisite.

Reply:
I agree - that was probably muscle memory starting with Ku and choosing poorly. :)

Reply:
did you forget to commit? 😅

Reply:
lol I think I changed the wrong one. I guess that's worse :laughing:

Reply:
![image](https://github.com/kubernetes-sigs/kueue/assets/814322/b1338bba-bfe8-4b0d-aecc-5df65c28ddb8)

Reply:
_I had one job..._